### PR TITLE
Add entity

### PR DESCRIPTION
**Pull request by CapMakse**

### CODE REVIEW

File: backend/src/main/java/com/eshka/entity/Queue.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    private long subject_id;
+}
+
+@Data
+public class Queue_details {
+    private long id;
+    private long queue_id;
+    private Timestamp turned_at;
+    private long user_id;
+}
```
Comment (TheBestTvarynka) [inline backend/src/main/java/com/eshka/entity/Queue.java]: Queue and Queue_details should be in separate files.
Java-file can't contain two public classes.

Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/entity/Queue.java]: Also, add annotations for constructors (`@NoArgsConstructor` and `@AllArgsConstructor`). [See documentation](https://projectlombok.org/features/constructor) for more.

File: backend/src/main/java/com/eshka/entity/Subject.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.eshka.entity;
+
+import lombok.Data;
+
+@Data
+public class Subject {
+    private long id;
+    private String name;
+}
```
Comment (TheBestTvarynka) [inline backend/src/main/java/com/eshka/entity/Subject.java]: Like in the comment above, pls, add constructor annotations.

File: backend/src/main/java/com/eshka/entity/Queue.java (hunk 1 above)
Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/entity/Queue.java]: Use `Data` type (from `java.utils.Data`) instead of `TimeStamp`.

**Review by TheBestTvarynka — CHANGES_REQUESTED** [review]
Please, read my comments and fix your code)
Happy coding!

File: backend/src/main/java/com/eshka/entity/Queue.java (hunk 1 above)
Comment (KySo4oK) [reply backend/src/main/java/com/eshka/entity/Queue.java]: > Use `Data` type (from `java.utils.Data`) instead of `TimeStamp`.

Data isn't deprecated?

Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/entity/Queue.java]: Hmm, I don't know. I will read it tonight.

**Review by KySo4oK — APPROVED** [review]

**Review by TheBestTvarynka — APPROVED** [review]
All good!
:moon_cake:

**Pull request merged**